models/line-item/src/line-item-draft/generator.ts

```diff
@@ -0,0 +1,34 @@
+import { Reference } from '@commercetools-test-data/commons';
+import { fake, Generator, oneOf } from '@commercetools-test-data/core';
+import * as Money from '@commercetools-test-data/money';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { inventoryMode } from '../constants';
+import { TLineItemDraft } from '../types';
+
+// https://docs.commercetools.com/api/projects/carts#lineitemdraft
+
+const [addedAt] = createRelatedDates();
+
+const generator = Generator<TLineItemDraft>({
+  fields: {
+    productId: fake((f) => f.datatype.uuid()),
+    sku: fake((f) => f.lorem.word()),
+    quantity: fake((f) =>
+      f.datatype.number({
+        min: 1,
+      })
+    ),
+    variantId: fake((f) => f.datatype.uuid()),
```

Comment:
These are typically numbers in a sequence, not UUID's

Reply:
Does this mean we can use our `sequence()` helper here as we do for `version` fields, or are you referring to a sequence of numbers only, ex `123-456-789` or similar?

Reply:
I think `sequence()` is completely appropriate here

Reply:
Awesome! Addressed in 1ebd471388033c9990801d10e1258b03e197753f